.github/workflows/main.yml

```diff
@@ -93,21 +93,24 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-
+      - name: Get dbt-core version
+        # if this is a pull request uses ref of base branch otherwise uses ref of current commit
+        id: dbt-core-version
+        run: echo "::set-output name=dbt-version::${{ github.event_name == 'pull_request' && github.base_ref || github.ref }}"
       - name: Install python dependencies
         run: |
           pip install  --user --upgrade pip
           pip install tox
           pip --version
           tox --version
 
-      - name: Install dbt-core latest
+      - name: Install dbt-core from branch ${{ steps.dbt-core-version.outputs.dbt-version }}
```

Comment:
Hmmm, I'm struggling with the rename here. "from branch?" I'm probably misunderstanding but is that needed here? I fee like what you're doing is installing the branch's dbt version as opposed to installing "from branch <some version sha>

Reply:
just a note!

`${{ steps.dbt-core-version.outputs.dbt-version }}` will be the name of the base branch of a PR (e.g. `1.0.latest`) or the name of the current branch for any other github event (manual trigger, push to main branch)

Reply:
maybe changing the name of `${{ steps.dbt-core-version.outputs.dbt-version }}` -> `${{ steps.dbt-core-version.outputs.dbt-core-ref }} might be more descriptive

Reply:
made name change to both main and integration ymls, thank you for the suggestion!